scout/build/variant/variant.py

```diff
@@ -199,7 +199,12 @@ def build_variant(
     variant_obj["str_pathologic_min"] = variant.get("str_pathologic_min")
     variant_obj["str_ref"] = variant.get("str_ref")
     variant_obj["str_repid"] = variant.get("str_repid")
+    variant_obj["str_trid"] = variant.get("str_trid")
```

Comment:
I think this kind of transformation should be done using a class, why not a Pydantic one since we have started using them. I would prefer to do that as a separate PR, knowing that we tend to introduce some issues with empty and missing values when we convert to Pydantic if it's ok with you.

Reply:
Ok!